### PR TITLE
feat:#507 update orders list frame html

### PR DESCRIPTION
**Pull request by Yingxue0323**

### CODE REVIEW

File: src/app/orders/page.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    const [data, detData] = useState([]);
+
+    return(
+    <div className="container">
+        <section className="section">
+            <div className="box">
+            <a className="tabs-more" href='/strategies'>'action.go_back'</a>
+            <div className="tabs">
+                <ul>
+                <li className="is-active"><a>'caption.investment_details'</a></li>
```
Comment (HuijunDev) [inline src/app/orders/page.tsx:15]: Please mind indentation.

**Review by HuijunDev — COMMENTED** [review]
Please ensure proper indentation and use two spaces instead of tabs to keep it consistent with the rest of the project.

File: src/app/orders/page.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import React, { useState, useEffect } from 'react';
+
+const orderList: React.FC = () => {
+    const [data, detData] = useState([]);
+
+    return(
+    <div className="container">
+        <section className="section">
+            <div className="box">
+            <a className="tabs-more" href='/strategies'>'action.go_back'</a>
```
Comment (ucfyao) [inline src/app/orders/page.tsx:12]: Follow our standards. You should maintain a uniform 2-space indentation format.

**Review by wjtwzgl12 — COMMENTED** [review]
All good!

File: src/app/orders/page.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
+'use client';
+
+import React, { useState, useEffect } from 'react';
+
+const orderList: React.FC = () => {
+    const [data, detData] = useState([]);
```
[the remark this answers is not in the document]
Comment (Yingxue0323) [reply src/app/orders/page.tsx]: > setData

Thanks, done.

**Pull request merged**